### PR TITLE
pkg/daemon/ceph/mon: add prefix to debug and cluster log streams to stderr

### PR DESCRIPTION
**Pull request by liewegas**
Debug log has "debug " prefix, cluster logs keep "default", "audit", etc.
prefixes (and now go to stderr).  This should allow both to be fed into the
logging stack (eg fluentd + elasticsearch or whatever) via stderr but allow the two
to be disambiguated by queries.

Note that this is in master, but backport for luminous has not merged yet, see

https://github.com/ceph/ceph/pull/19849

### CODE REVIEW

**Comment by galexrt** [comment]
@liewegas From what I see the Ceph backport for Luminous been merged. Is this ready now?
(Rook has already picked up `ceph version 12.2.4 (52085d5249a80c5f5121a76d6288429f35e4e77b) luminous (stable)` in release 0.7.1)

Could you rebase the PR and also add DCO commit sign-off, see https://github.com/rook/rook/blob/master/CONTRIBUTING.md#certificate-of-origin.

**Comment by liewegas** [comment]
Yep, should be good to merge!

**Comment by galexrt** [comment]
@liewegas Could you please add the sign off to your commit

**Comment by liewegas** [comment]
I'm confused.. it's always been there?

**Comment by galexrt** [comment]
Oh, the DCO check was red for me just a few minutes ago.. Well now it's green :)

**Comment by galexrt** [comment]
@liewegas Could you please rebase the PR to see if CI will be green now.

File: pkg/daemon/ceph/mon/daemon.go [hunk 1]
```diff
@@ -128,6 +128,7 @@ func startMon(context *clusterd.Context, config *Config, confFilePath, monDataDi
 		"--mkfs",
 		monNameArg,
 		fmt.Sprintf("--cluster=%s", config.Cluster.Name),
+		fmt.Sprintf("--log-stderr-prefix=debug "),
```
Comment (galexrt) [inline pkg/daemon/ceph/mon/daemon.go]: If that is `stderr` wouldn't the prefix `error` be more fitting?

**Comment by travisn** [comment]
Based on the integration test failures, I suspect the change in logging is causing the rgw service, or some other part of the orchestration to fail. The [operator log](https://jenkins.rook.io/job/rook/job/rook/job/PR-1388/5/artifact/_output/tests/TestSmokeSuite_aws_1.7.x_rook-ceph-operator-3995327236-t5qsr_smoke-ns-system_1529429785.logj) shows at the end that the mons started going out of quorum. It's not obvious to me in the mon logs (see the [artifacts](https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/PR-1388/5/artifacts)) what the issue was.

File: pkg/daemon/ceph/mon/daemon.go (hunk 1 above)
Comment (liewegas) [reply pkg/daemon/ceph/mon/daemon.go]: this is making the debug log (what normally goes to e.g., /var/log/ceph/ceph-mon.foo.log) to go stderr.  The goal is to disambiguate this from the cluster logs (cluster, audit, etc) that normally go to /var/log/ceph/ceph.log or ceph.audit.log

**Comment by liewegas** [comment]
rebased...

**Comment by leseb** [comment]
This should be good to go after one more rebase, right?

**Comment by travisn** [comment]
This is being included with #2462

**Pull request closed without merging**